bigbluebutton-html5/imports/ui/components/audio/permissions-overlay/component.jsx

```diff
@@ -34,6 +34,10 @@ class PermissionsOverlay extends Component {
         top: '100px',
         left: '100px',
       },
+      'Microsoft Edge': {
+        top: '38em',
```

Comment:
I believe it's not a good idea to use "EM" here, when the browser is in half screen the overlay does not appear
![bug-overlay](https://user-images.githubusercontent.com/15806883/41724759-a3ca5fac-7544-11e8-8639-716d1c16736f.png)

Reply:
I can change it to pixels, but it wont fix this problem. The audio-overlay is not responsive, on any of the browsers.

Reply:
`top: 'unset', bottom: 'Xpx'`